core/renderers/common/renderer.js

```diff
@@ -206,11 +204,10 @@ Renderer.prototype.makeMarkerDrawer = function(
  * @return {!IPathObject} The renderer path object.
  * @package
  */
-Renderer.prototype.makePathObject = function(root,
-    style) {
-  return new PathObject(root, style,
+Renderer.prototype.makePathObject = function(root, style) {
+  return new PathObject(
+      root, style,
       /** @type {!ConstantProvider} */ (this.constants_));
```

Comment:
Will clang-format let you remove the line break after `style,`?

Reply:
Yes, it lets me change that. I've also updated another line to remove the line break.